sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/cryptography/cryptography_client.hpp

```diff
@@ -9,7 +9,7 @@
 
 #pragma once
 
-#include <azure/keyvault/common/internal/keyvault_pipeline.hpp>
+#include "../src/private/keyvault_protocol.hpp"
```

Comment:
The protocol client will be required for the other packages. Now that @gearama  is adding Secrets, he will use the protocolClient.
So, do not move it to Keys. Instead, leave the common folder and the file in there. We need to make CMake to copy the file to Keys, or use it from that path to build the package.

Reply:
We discussed this offline yesterday, and doing this is the smallest change to defer the larger refactoring in the future while removing the dependency on common for now. We can decide what we want to do in the future, because it is currently an implementation detail.